src/Wallabag/CoreBundle/Controller/WallabagRestController.php

```diff
@@ -131,17 +165,18 @@ public function postEntriesAction(Request $request)
      *          {"name"="tags", "dataType"="string", "required"=false, "format"="tag1,tag2,tag3", "description"="a comma-separated list of tags."},
      *          {"name"="archive", "dataType"="boolean", "required"=false, "format"="true or false", "description"="archived the entry."},
      *          {"name"="star", "dataType"="boolean", "required"=false, "format"="true or false", "description"="starred the entry."},
-     *          {"name"="delete", "dataType"="boolean", "required"=false, "format"="true or false", "description"="flag as deleted. Default false. In case that you don't want to *really* remove it.."},
-     *       }
+     *      }
      * )
      * @return Entry
      */
     public function patchEntriesAction(Entry $entry, Request $request)
     {
+        if ($entry->getUser()->getId() != $this->getUser()->getId()) {
+            throw $this->createAccessDeniedException();
```

Comment:
About those `createAccessDeniedException`, maybe you can add a message to tell more about the denied access, like `entry user id: blabla, logged user id: blabla`. It will more easy to debug later :)